src/main/export.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-plusplus */
+import { join } from 'path';
+import { Project, Transcription } from '../sharedTypes';
+import { padZeros, integerDivide, writeFile, mkdir } from './util';
+
+const secondToTimestamp = (num: number) => {
+  return `${padZeros(integerDivide(num, 3600), 2)}:
+          ${padZeros(integerDivide(num, 60), 2)}:
+          ${padZeros(integerDivide(num, 1), 2)}:
+          ${padZeros(integerDivide(num, 0.01), 2)}`;
+};
+
+const constructEDL = (title: string, transcription: Transcription) => {
```

Comment:
This is a good candidate for some unit testing, fine to do this in a later PR, would you be confident in adding some tests with jest?

Reply:
I can try

Reply:
Great, feel free to reach out (to myself or anyone else familiar with jest) if you need a hand